scoper.inc.php

```diff
@@ -10,11 +10,27 @@
 
 use Isolated\Symfony\Component\Finder\Finder;
 
+/**
+ * Gets a list of WP globals to ignore.
+ *
+ * @param string $json_file the file name to pull the excludes list from.
+ * @return array[] Array of WP Classes, WP Functions, or WP Constants to exclude.
+ */
+function get_excludes( string $json_file ): array {
+	try {
+		$excludes_vendor_path = 'vendor/sniccwp/php-scoper-wordpress-excludes/generated/';
+		return json_decode( file_get_contents( $excludes_vendor_path . $json_file ) );
+	} catch ( \Exception $exception ) {
```

Comment:
Was there a lint warning or something about this?

The simple `json_decode( file_get_contents( 'vendor/sniccwp/php-scoper-wordpress-excludes/generated/exclude-wordpress-classes.json' ) );` statements you had originally seemed fine to me

Reply:
I can take it back out if you like ... was looking a little repetitive + getting long with the array merge.

Reply:
We can discuss it today but it didn't feel like a big issue at all.

Reply:
reverted to the get contents statements + updated to pull from the `sniccowp/php-scoper-wordpress-excludes` package name